src/components/Layout/Layout.module.css

```diff
@@ -0,0 +1,87 @@
+.newLayout {
+  --black: lab(0.07 0.35 0.12);
+  --dark-grey: lab(43.19 0 0);
+  --grey: lab(66.99 0 0);
+  --light-grey: lab(94.10 0 0);
+  --off-white: lab(100 0 0);
+
+  --blue: lab(49.827 31.172 -78.866);
+
+  --background-color: var(--light-grey);
+  --text-color: var(--black);
+  --link-color: var(--dark-grey);
+
+  --wrapper-width: 1192px;
+
+  &.isAfterDark {
+    --background-color: var(--black);
+    --text-color: var(--off-white);
+    --link-color: var(--grey);
+  }
+}
+
+.newLayout {
+  @mixin body-4;
+  background-color: var(--background-color);
+  color: var(--text-color);
+  width: 100%;
+  padding: 0 20px;
+
+  @media (--tablet) {
+    @mixin body-2;
+    padding: 0 40px;
+  }
+
+  & .newLayout {
+    padding: 0;
+  }
+
+  & :global(a) {
+    position: relative;
+    color: var(--link-color);
+    text-decoration-clor: currentColor;
+    text-decoration-thickness: 1px;
+    text-underline-position: from-font;
+  }
+
+  & .wrapper {
+    padding-top: 268px;
+    padding-bottom: 104px;
+    max-width: 1192px;
+    margin: 0 auto;
+
+    @media (--desktop) {
+      padding-top: 304px;
+      padding-bottom: 228px;
+    }
+  }
+}
+
+.oldLayout {
+  --blue: lab(49.827 31.172 -78.866);
+
+  & .newLayout {
+    background-color: var(--white);
+  }
+}
+
+
+.transitionColorFlash {
+  background: var(--blue);
```

Comment:
No longer a random color, just always blue? :'(

Reply:
Designers don't like it.

Reply:
Larix, reviewed this at 12AM?